CHANGELOG.md

```diff
@@ -3,6 +3,62 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 6.12.1
+
+_Aug 31, 2023_
+
+We'd like to offer a big thanks to the 7 contributors who made this release possible. Here are some highlights ✨:
+
+- 🏎️ Perf improvement for line charts
```

Comment:
Maybe also the `referenceDate` addition as it's a pretty significant addition. 🤔 
```suggestion
- 🏎️ Perf improvement for line charts
- 🎁 Add `referenceDate` prop on pickers (#9991) @flaviendelangle
  Find out more about this feature in the [documentation section](https://mui.com/x/react-date-pickers/base-concepts/#reference-date-when-no-value-is-defined).
```